src/network/generator/network-generator.c

```diff
@@ -69,10 +70,29 @@ static const char * const networkd_dhcp_type_table[_DHCP_TYPE_MAX] = {
         [DHCP_TYPE_EITHER6] = "ipv6", /* TODO: enable other setting? */
         [DHCP_TYPE_IBFT]    = "no",
         [DHCP_TYPE_LINK6]   = "no",
+        [DHCP_TYPE_LL]      = "no",
 };
 
 DEFINE_PRIVATE_STRING_TABLE_LOOKUP_TO_STRING(networkd_dhcp_type, DHCPType);
 
+static const char * const networkd_ipv6ra_type_table[_DHCP_TYPE_MAX] = {
+        [DHCP_TYPE_NONE]    = "no",
+        [DHCP_TYPE_OFF]     = "no",
+        [DHCP_TYPE_LL]      = "no",
+        /* We omit the other entries, to leave the default in effect */
+};
+
+DEFINE_PRIVATE_STRING_TABLE_LOOKUP_TO_STRING(networkd_ipv6ra_type, DHCPType);
+
+static const char * const networkd_link_local_type_table[_DHCP_TYPE_MAX] = {
+        [DHCP_TYPE_NONE]    = "no",
+        [DHCP_TYPE_OFF]     = "no",
```

Comment:
I do not think we should disable IPv6LL in these two cases. Why do you disable it?

Reply:
well, dunno, it sounded to me that the user really doesn't want automatic stuff if they say ip=no? do you think that goes too far? "ip=no" to me sounds like a very harsh way to say "no matter what, do not give me IP", so I figured we turn ipv6ll off too.

Admittedly this requires a NEWS change, since it is a compat break on some level. Does that make sense to you, if we keep this but I already prep a NEWS entry?

Reply:
OK, by reconsidering about that, we already have link6. So the change makes sense to me.